client/signup/steps/jpo-homepage/style.scss

```diff
@@ -0,0 +1,206 @@
+.jpo__homepage-wrapper {
+	margin-left: auto;
+	margin-right: auto;
+	display: flex;
+}
+
+.jpo__homepage-row {
+	margin: 0 auto;
+	display: flex;
+	flex-flow: row wrap;
+	max-width: 640px;
+
+	.card {
+		width: 46%;
+		display: inline-block;
+		vertical-align: top;
+		text-align: center;
+		padding: 0;
+
+		@include breakpoint( "<480px" ) {
+		width: 100%;
+		text-align: left;
+		}
+
+		&:hover,
+		&.is-selected {
+			box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20 );
```

Comment:
We should make sure the borders and shadows in this component match jpo-site-type/style.scss after my suggestions have been implemented. Even better if we make most of this stuff into reusable components.